buildkite/src/Command/OpamInit.dhall

```diff
@@ -0,0 +1,30 @@
+let Prelude = ../External/Prelude.dhall
+
+let Cmd = ../Lib/Cmds.dhall
+
+let r = Cmd.run
+
+let commands : List Cmd.Type =
+  [
+    r "cat scripts/setup-opam.sh > opam_ci_cache.sig",
+    r "cat src/opam.export >> opam_ci_cache.sig",
```

Comment:
You can shorten this as `cat scripts/setup-opam.sh src/opam.export > opam_ci_cache.sig`. If you want, you can even fold the `date` portion of the cache signature into this command as well with a `<()`.